src/metabase/db/custom_migrations/metrics_v2.clj

```diff
@@ -0,0 +1,158 @@
+(ns metabase.db.custom-migrations.metrics-v2
+  (:require
+   [cheshire.core :as json]
+   [metabase.util :as u]
+   [metabase.util.log :as log]
+   [toucan2.core :as t2])
+  (:import (java.time Instant)))
+
+(set! *warn-on-reflection* true)
+
+(defn- get-metric-migration-collection
+  "Get the collection used to store the questions created from the v1 metrics.
+  If and only if the collection doesn't exists and the option `create?` is truthy,
+  the collection is created."
+  ([]
+   (get-metric-migration-collection nil))
+  ([{:keys [create?]}]
+   (let [coll-name "Migrated Metrics v1"
+         slug (u/slugify coll-name 510)
```

Comment:
I'm sure that `510` comes from somewhere, but I have no idea where -- probably worth explaining why you picked that in the comments. Also FWIW slugifying `Migrated Metrics v1` is deterministic and always returns `migrated_metrics_v1`, so there is no universe where that's going to be more than 510 bytes.

I think it would be better just to hardcode the slugified name here and remove the call to `u/slugify`, that way on the off chance that someone changes `u/slugify` in the future it doesn't break downward migrations here